cmd/rosa-support/create/sg/cmd.go

```diff
@@ -51,7 +51,7 @@ func init() {
 		&args.count,
 		"count",
 		"",
-		0,
+		1,
```

Comment:
Is this change expected?

Reply:
if you do not set `--count` yourself, QE does not have this required in their normal flow though